### PR TITLE
Primary Class Syntax

### PR DESCRIPTION
**Pull request by FlXlT**
This PR includes the implementation to make the Class definition syntax work. This also means a lot of different areas have been implemented, some not fully. For example, the method definition/invocation was partly implemented to make the body statement of a class work.

As an addition this PR includes a test suite for the Class definition.

### CODE REVIEW

File: ruby.example/classes/Simple.aterm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Program(
```
Comment (jessetilro) [inline ruby.example/classes/Simple.aterm]: I would remove this file and make sure all aterm files are ignored.

File: ruby.example/fragments/compount_statement.aterm [hunk 2]
```diff
@@ -1,3 +1,6 @@
 Program(
```
Comment (jessetilro) [inline ruby.example/fragments/compount_statement.aterm]: Remove this file

**Review by jessetilro — COMMENTED** [review]
Nice progress. Just added a few comments about files to be removed.

**Comment by FlXlT** [comment]
Fixed, see #9

**Pull request merged**